Ktisis/History/HistoryManager.cs

```diff
@@ -131,9 +131,8 @@ public static class HistoryManager {
 				PluginLog.Warning("Attempted to add an entry to an uninitialised history list.");
 				return;
 			}
-			History.Insert(_maxIdx, historyItem);
+			History.Insert(History.Count, historyItem);
```

Comment:
Use `History.Add` instead.
```suggestion
			History.Add(historyItem);
```